resources/js/pages/users/edit.tsx

```
@@ -11,26 +11,27 @@ import { router } from '@inertiajs/react';
 import { useForm } from '@inertiajs/react';
 import { toast } from 'react-hot-toast';
 import { cn } from '@/lib/utils';
-import { RolePermission } from '@/components/role-permission';
+import { UserRoleSelect } from '@/components/user-role-select';
+import { RolePermissionManager } from '@/pages/users/partials/role-permission-manager';
```

ESLint

[error] 'RolePermissionManager' is defined but never used.

(@typescript-eslint/no-unused-vars)